docs/src/main/sphinx/release/release-382.md

```diff
@@ -0,0 +1,99 @@
+# Release 382 (25 May 2022)
+
+## General
+
+* Improve join performance. ({issue}`12257`)
+* Improve coordinator memory management for task-based fault-tolerant execution.
+  This decreases the chance of queries failing with the
```

Comment:
@losipiuk  ... what do you think.. could we just remove the sentence here?

Reply:
Depending on how explicit we want to be in RN. With the extra sentence, it is easier for users to understand if the new release will solve the problems they are seeing or not. But I do not feel strongly.

Reply:
I would rephrase as:

> Fix potential query failures due to `EXCEEDED_TASK_DESCRIPTOR_STORAGE_CAPACITY` errors when executing queries with `TASK` retry-policy